.github/workflows/report.yml

```diff
@@ -0,0 +1,55 @@
+name: Status Report
+
+on:
+  workflow_call:
+    secrets:
+      ALTERNATIVE_GITHUB_TOKEN:
+        required: false
+
+jobs:
+
+  report:
+    runs-on: ubuntu-22.04
+    env:
+      GITHUB_TOKEN: ${{ secrets.ALTERNATIVE_GITHUB_TOKEN ||  secrets.GITHUB_TOKEN }}
+
+    steps:
+      - name: SCM Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Setup Python & Poetry Environment
+        uses: exasol/python-toolbox/.github/actions/python-environment@0.20.0
+
+      - name: Download Artifacts
+        uses: actions/download-artifact@v4.1.8
+        with:
+          path: ./artifacts
+
+      - name: Copy Artifacts into Root Folder
+        working-directory: ./artifacts
+        run: |
+          poetry run coverage combine --keep coverage-python3.9*/.coverage
```

Comment:
This is probably right, but why is it fixed on python3.9?

Reply:
Some jobs shall only run with one specific python version, currently this is python 3.9 ideally in the future this will be made also configureable.